javascript/ql/test/library-tests/Security/heuristics/sources.js

```diff
@@ -152,6 +153,7 @@ function sink(data) {}
         txn.run('SELECT * FROM users', function (err, users) {
             sink(users); // NOT OK
         });
+        txn.commit(function () {});
```

Comment:
Why has this line been added to the test? 
Was it to make the code in the test more realistic? (a perfectly valid reason).

Reply:
yes, realistic!